.env.example

```
@@ -1,34 +1,64 @@
 # HubSpot Signal-First Account Workspace
-# Copy to .env and fill in values
+# Copy to .env and fill in values.
 # ===========================================
 
 # --- Database ---
+# Local dev: Docker Postgres on :5433 (see docker-compose.yml).
+# Production tenants set DATABASE_URL to their own managed Postgres
+# (Supabase / Neon / RDS / etc.) — the product is config-driven.
 DATABASE_URL=postgresql://hap:hap_local_dev@localhost:5433/hap_dev
 POSTGRES_USER=hap
 POSTGRES_PASSWORD=hap_local_dev
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)